### ISSUE TITLE
Add gitignore file

### ISSUE THREAD

**Issue by km4** [schedutron/home-server#30]
We should make gitignore file.

I create the PR with this issues #29

**Comment by schedutron** [comment]
Indeed! By the way, I already have a local .gitignore. I hope there are no merge conflicts with this one.

**Comment by mayankagarwal2402** [comment]
hi @schedutron. First time OSS contributor here. I would like to volunteer for this issue.

**Comment by helderberto** [comment]
Updated .gitignore #54.

**Comment by shalika10** [comment]
hey is it still open ?
